contrib/python/api/skydive/wsshell.py

```diff
@@ -141,7 +142,11 @@ def main():
                       mode=args.mode,
                       file=file)
     client.connect()
+    client.start()
 
 
 if __name__ == '__main__':
+    logging.basicConfig()
+    WSClient.LOG.setLevel(logging.DEBUG)
```

Comment:
I think you can't access to the LOG variable without importing it

Reply:
We have imported the entire package:
from skydive.websocket.client import WSClient (line 28)
so, everything in it is accessible, there is no need to import it explicitly,

Reply:
mm the class is imported not the whole package ? get the following error running it

```
Traceback (most recent call last):
  File "skydive/wsshell.py", line 150, in <module>
    WSClient.LOG.setLevel(logging.DEBUG)
AttributeError: type object 'WSClient' has no attribute 'LOG'
```

Reply:
Sorry, my bad. You are right. Will fix.